docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -647,6 +695,7 @@
                         The fully-qualified name of the Java class which determines the factory class which will build the TaskNameGrouper.
                         The default configuration value if the property is not present is <code>task.name.grouper.factory=org.apache.samza.container.grouper.task.GroupByContainerCountFactory</code>.<br>
                         The user can specify a custom implementation of the TaskNameGrouperFactory where a custom logic is implemented for grouping the tasks.
+                    <p><strong> Note.</strong> For StandAlone Application one must use <i>org.apache.samza.container.grouper.task.GroupByContainerIdsFactory</i>
```

Comment:
"Note:"

Shouldn't use standalone as the feature name. What do you mean by StandAlone Application here? Ones using StandaloneJobCoordinatorFactory? If so, let's clarify.